src/components/Table/GridTable.tsx

```diff
@@ -558,6 +575,52 @@ function renderVirtual<R extends Kinded>(
   );
 }
 
+/**
+ * renderVirtual without the `display: contents`
+ */
+function renderVirtualFixed<R extends Kinded>(
```

Comment:
@blambillotte we made it work by taking off the `display: contents` which now is removed completely for virtual tables. So no need to create a new `as` type and you can ignore the majority of this code.